package.json

```diff
@@ -26,7 +26,7 @@
     "govuk_template_jinja": "^0.22.2",
     "nunjucks": "^3.0.1",
     "passport": "^0.3.2",
-    "passport-verify": "0.8.0",
+    "passport-verify": "^0.8.0",
```

Comment:
yarn.lock should be the file used to version lock our deps so let's use the most up-to-date here.